RadarSDK/RadarLocationManager.m

```diff
@@ -304,17 +304,18 @@ - (void)updateTracking:(CLLocation *)location fromInitialize:(BOOL)fromInitializ
     dispatch_async(dispatch_get_main_queue(), ^{
         BOOL tracking = [RadarSettings tracking];
         RadarTrackingOptions *options = [Radar getTrackingOptions];
+        RadarTrackingOptions *localOptions = [RadarSettings trackingOptions];
 
         [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug
                                            message:[NSString stringWithFormat:@"Updating tracking | options = %@; location = %@", [options dictionaryValue], location]];
 
-        if (!tracking && [options.startTrackingAfter timeIntervalSinceNow] < 0) {
+        if (!tracking && [localOptions.startTrackingAfter timeIntervalSinceNow] < 0) {
```

Comment:
I understand that we are implementing this to support scheduled trips in the future. But say we schedule the trip a few days from now, what would trigger our SDK attempting to `updateTracking` and actually start tracking at the scheduled time?

Reply:
also discussed irl. `startTrackingAfter` requires a `trackOnce` to indeed actually start tracking.